src/awsLambda.re

```diff
@@ -495,3 +495,97 @@ module Sqs = {
 
   type handler = handler_error(Event.t);
 };
+
+module S3 = {
+  module UserIdentity = {
+    [@bs.deriving abstract]
+    type t = {principalId: string};
+
+    let make = t;
+  };
+
+  module RequestParameters = {
+    [@bs.deriving abstract]
+    type t = {sourceIPAddress: string};
+
+    let make = t;
+  };
+
+  module ResponseElements = {
+    [@bs.deriving abstract]
+    type t = {
+      [@bs.as "x-amz-request-id"]
+      xAmzRequestId: string,
+      [@bs.as "x-amz-id-2"]
+      xAmzId2: string,
+    };
+
+    let make = t;
+  };
+
+  module S3Info = {
```

Comment:
This module could be named `Info` rather than `S3Info` as it is already in the `S3` module.